cfme/services/catalogs/catalog_item.py

```diff
@@ -204,7 +204,7 @@ def create(self):
                    'description': self.description,
                    'display': self.display_in,
                    'select_catalog': self.catalog.name,
-                   'select_dialog': self.dialog,
+                   'select_dialog': self.dialog.label,
```

Comment:
Once you implement the as_fill_value, you can drop the `.label` because it will know how it is displayed in the UI.